kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/rest/IdentifierJbpmRestIntegrationTest.java

```diff
@@ -0,0 +1,133 @@
+import static org.kie.server.api.rest.RestURI.START_PROCESS_POST_URI;
+import static org.kie.server.api.rest.RestURI.build;
+
+
+public class IdentifierJbpmRestIntegrationTest extends RestJbpmBaseIntegrationTest {
+
+
+    private static ReleaseId releaseId = new ReleaseId("org.kie.server.testing", "rest-processes", "1.0.0.Final");
+   
+    private static Logger logger = LoggerFactory.getLogger(IdentifierJbpmRestIntegrationTest.class);
+
+
+    @BeforeClass
+    public static void buildAndDeployArtifacts() {
+        KieServerDeployer.buildAndDeployCommonMavenParent();
+        KieServerDeployer.buildAndDeployMavenProjectFromResource("/kjars-sources/rest-processes");
+
+        createContainer(CONTAINER, releaseId);
+
+    }
+
+
+    private static final String CONTAINER = "rest-processes";
+    private static final String HUMAN_TASK_OWN_TYPE_ID = "org.test.kjar.HumanTaskWithOwnType";
+
+
+    @Test
+    public void testBasicJbpmRequest() throws Exception {
+        KieContainerResource resource = new KieContainerResource(CONTAINER, releaseId);
+
+        Map<String, Object> valuesMap = new HashMap<String, Object>();
+        valuesMap.put(CONTAINER_ID, resource.getContainerId());
+        valuesMap.put(PROCESS_ID, HUMAN_TASK_OWN_TYPE_ID);
+
+        Response response = null;
+        try {
+            WebTarget clientRequest = newRequest(build(TestConfig.getKieServerHttpUrl(), PROCESS_URI + "/" + START_PROCESS_POST_URI, valuesMap));
+            logger.info( "[POST] " + clientRequest.getUri());
+            // we set strict json parameter
+            MediaType type = getMediaType();
+            type = new MediaType(type.getType(), type.getSubtype(), Collections.singletonMap("strict", "true"));
+            logger.info("media type : {}", type);
+            
+            response = clientRequest.request(type).post(createEntity(""));
+            Assert.assertEquals(Response.Status.CREATED.getStatusCode(), response.getStatus());
+            Assertions.assertThat((String)response.getHeaders().getFirst("Content-Type")).startsWith(getMediaType().toString());
+
+            String serialized = response.readEntity(String.class);
+            response.close();
+            Marshaller marshaller = MarshallerFactory.getMarshaller(marshallingFormat, Thread.currentThread().getContextClassLoader());
+            Object wrapId = null;
+            Object id = null;
+            logger.info("Value returned {} {}", serialized, getMediaType());
+            if (marshallingFormat.equals(MarshallingFormat.JAXB)) {
+                wrapId = marshaller.unmarshall(serialized, Long.class);
+                id = ((Long) wrapId);
+            } else {
+                try {
+                    wrapId = new JSONObject(serialized);
+                    id = ((JSONObject) wrapId).getString("value");
+                } catch (JSONException ex) {
+                    logger.error("expected json object from kie server", ex);
```

Comment:
Please don't log errors in test, rather throw it making the test fail.

Reply:
done